tests/process/spawn.luau

```diff
@@ -91,10 +106,22 @@ assert(homeDir1 == homeDir2, "Home dirs did not match when performing tilde subs
 	than a single sleep but also less than 1.5 sleeps
 ]]
 
+-- We need to account for the time it takes to spawn a process,
+-- due to Windows having a delay in launching processes.
+-- Windows delay is likely caused by Windows Defender.
+local win_commandExecutionTolarance = 0 do
+	if isWindows then
+		local commandExecutionStart = os.clock()
+		process.spawn("sleep", {"0"}, if isWindows then { shell = true } else nil)
+		win_commandExecutionTolarance = os.clock() - commandExecutionStart
+	end
+end
+
```

Comment:
This feels a bit magical and CI shouldn't be running anything like Windows Defender (but I haven't checked), is there any other way we could do this? Would a small additional allowed amount of time work as well?

Reply:
To add on to the above, trying to have precise timings on Windows is notoriously bad (see `task.wait` tests...), so it's probably fine to just not test whether `sleep` is precise or not on Windows if we don't find a good solution.

Reply:
Even if CI doesn't run windows defender, it is possible for new users that test on windows could face Windows defender problems. Not exactly sure what would be the best way to account for those.

Reply:
With that large of a variance (0.5 vs 1.9s) it's probably better to skip the part where we check sleep on Windows. 😕 We can make it into more of a timeout thing instead, so if 15 seconds or so have passed, it more than likely just failed, and we force exit the test

Reply:
We could possibly just run a sleep command with a 1 second time, and check if the command time was at least 1 second long and less than 15 seconds like you suggested.

Reply:
I have tested the new 15 seconds changes, it still seems to short on CI windows test., check it out on my [run](https://github.com/SnorlaxAssist/lune/actions/runs/6302046775/job/17108431890).

Reply:
I have found out after a little bit more digging, that it takes 14-15 seconds for CI to run sleep command in windows, maybe 25 could be the better option?